
https://github.com/stefanesser/suhosin/issues/42

--- suhosin-0.9.35/php_suhosin.h~	2014-02-25 00:39:47.000000000 +0200
+++ suhosin-0.9.35/php_suhosin.h	2014-04-28 15:07:18.035583796 +0300
@@ -286,10 +286,11 @@
 #define S_MAIL				(1<<7L)
 #define S_SESSION			(1<<8L)
 #define S_INTERNAL			(1<<29L)
-#define S_GETCALLER         (1<<30L)
 #define S_ALL (S_MEMORY | S_VARS | S_INCLUDE | S_FILES | S_MAIL | S_SESSION | S_MISC | S_SQL | S_EXECUTOR)
 #endif
 
+#define S_GETCALLER         (1<<30L)
+
 #define SUHOSIN_NORMAL	0
 #define SUHOSIN_EVAL	1
 
